pkg/kubefed2/federate/util.go

```diff
@@ -49,3 +59,134 @@ func SetBasicMetaFields(resource *unstructured.Unstructured, apiResource metav1.
 		resource.SetNamespace(namespace)
 	}
 }
+
+func namespacedAPIResourceMap(config *rest.Config, skipAPIResourceNames string) (map[string]metav1.APIResource, error) {
+	apiResourceLists, err := enable.GetServerPreferredResources(config)
+	if err != nil {
+		return nil, err
+	}
+
+	apiResources := make(map[string]metav1.APIResource)
```

Comment:
The keys of this map should be `<group-qualified plural name>` rather than kind, since kind is not guaranteed to be globally unique.

Reply:
Please see [this comment](https://github.com/kubernetes-sigs/federation-v2/pull/741#issuecomment-488247153).